ipa-core/src/protocol/basics/mul/mod.rs

```diff
@@ -21,6 +21,7 @@ pub(crate) mod malicious;
 mod semi_honest;
 pub(in crate::protocol) mod sparse;
 
+#[allow(unused_imports)]
```

Comment:
why do you need that?

Reply:
clippy was complaining, some of the github runs failed without it

Reply:
most of the time it complains about legit issues - if we don't use this multiplication anymore, we shouldn't export it.

this suppression would make more sense to me as temporary measure

Reply:
I think it is related to the fact that we only use it when running maliciously secure IPA. In semi honest IPA, it is not used.